corehq/apps/commtrack/models.py

```diff
@@ -389,61 +391,8 @@ def balance(self):
         return self.stock_on_hand
 
     @property
-    def months_remaining(self):
-        return months_of_stock_remaining(
-            self.stock_on_hand,
-            self.get_daily_consumption()
-        )
-
-    @property
-    def resupply_quantity_needed(self):
-        monthly_consumption = self.get_monthly_consumption()
-        if monthly_consumption is not None and self.sql_location is not None:
-            overstock = self.sql_location.location_type.overstock_threshold
-            needed_quantity = int(
-                monthly_consumption * overstock
-            )
-            return int(max(needed_quantity - self.stock_on_hand, 0))
-        else:
-            return None
-
-    @property
-    def stock_category(self):
-        return state_stock_category(self)
-
-    @memoized
-    def get_domain(self):
-        return Domain.get_by_name(
-            DocDomainMapping.objects.get(doc_id=self.case_id).domain_name
-        )
-
-    def get_daily_consumption(self):
-        if self.daily_consumption is not None:
-            return self.daily_consumption
-        else:
-            monthly = self._get_default_monthly_consumption()
-            if monthly is not None:
-                return Decimal(monthly) / Decimal(DAYS_IN_MONTH)
-
-    def get_monthly_consumption(self):
-
-        if self.daily_consumption is not None:
-            return self.daily_consumption * Decimal(DAYS_IN_MONTH)
-        else:
-            return self._get_default_monthly_consumption()
-
-    def _get_default_monthly_consumption(self):
-        domain = self.get_domain()
-        if domain and domain.commtrack_settings:
-            config = domain.commtrack_settings.get_consumption_config()
-        else:
-            config = None
-
-        return compute_default_monthly_consumption(
-            self.case_id,
-            self.product_id,
-            config
-        )
+    def domain(self):
+        return DocDomainMapping.objects.get(doc_id=self.case_id).domain_name
```

Comment:
That's how it's working now. It must work otherwise we'd see a lot of `DocDomainMapping.DoesNotExist` errors (which is what you get in tests if mess up the signals).

I'd love to get rid of this though. Do you know more about it's origin and why it was created?